src/main/java/com/google/sps/servlets/MutableHttpServletRequest.java

```diff
@@ -1,3 +1,5 @@
+package com.google.sps.servlets;
+
```

Comment:
Just curious why do we introduce this?

Reply:
When the auth-token is verified, it gives us a userID. So we need to add that userID to the request as a header
HttpServletRequest doesn't have a addHeader() method so this class creates a HttpServletRequest that we can add the header to. 
https://wilddiary.com/adding-custom-headers-java-httpservletrequest/